src/ol/observable.js

```diff
@@ -62,6 +62,18 @@ ol.Observable.prototype.changed = function() {
 
 
 /**
+ * Dispatches an event (or event like object) and calls all listeners
+ * listening for events of this type. The type of the event is decided by the
+ * type property on the event object.
+ *
+ * @param {goog.events.EventLike} event Event object.
+ * @function
+ * @api
+ */
+ol.Observable.prototype.dispatchEvent;
```

Comment:
I think it should look like this:

```js
ol.Observable.prototype.dispatchEvent = goog.events.EventTarget.prototype.dispatchEvent;
```

And also, we don't want to use `@param {goog.events.EventLike}`, to avoid exposing `goog` types in the API doc. So I think we need to use `@param {string}`, although this is a bit limitating.